Makefile.PL

```diff
@@ -203,6 +203,8 @@ sub install {
     my ($self) = @_;
     my $install = $self->SUPER::install(@_);
 
+    return $install if $OSNAME eq 'MSWin32';
```

Comment:
It causes a fail test in AppVeyor environment when 'make' is launched. I didn't think about Cygwin